### PR TITLE
PORTALS-3001

### PR DESCRIPTION
**Pull request by jay-hodgson**

### CODE REVIEW

File: apps/portals/src/configurations/genie/resources.ts [hunk 1]
```diff
@@ -1,9 +1,6 @@
-export const programSql = 'SELECT * FROM syn36649342'
-export const projectsSql = 'SELECT * FROM syn36649647'
-export const datasetsSql = 'SELECT * FROM syn39041190'
-
 export const contributorsSql = 'SELECT * FROM syn53975138'
 export const genieBPCSql = 'SELECT * FROM syn38806899'
 export const genieDataSql = 'SELECT * FROM syn38806899'
 export const publicationsSql = 'SELECT * FROM syn53974927'
 export const toolsSql = 'select * from syn53975155'
+export const dataReleasesSql = 'select * from syn53977725'
```
Comment (jay-hodgson) [inline apps/portals/src/configurations/genie/resources.ts:6]: Add a new Data Releases page, that currently has a table of all data releases (hidden in a new Collapsable wrapper)

File: apps/portals/src/configurations/genie/routesConfig.ts [hunk 2]
```diff
@@ -73,6 +74,11 @@ const routes: GenericRoute[] = [
       },
     ],
   },
+  {
+    path: 'Data Releases',
+    exact: true,
+    synapseConfigArray: dataReleases,
+  },
```
Comment (jay-hodgson) [inline apps/portals/src/configurations/genie/routesConfig.ts:81]: Screen Shot 2024-03-14 at 2 45 24 PM

File: apps/portals/src/configurations/genie/synapseConfigs/dataReleases.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  {
+    name: 'Markdown',
+    title: 'Previous Data Release',
+    props: {
+      markdown:
+        'The **latest** version of the dataset is readily acccessible for free download, but access to **previous** versions must be requested.',
+    },
+  },
+  {
+    name: 'SynapseComponentCollapse',
```
Comment (jay-hodgson) [inline apps/portals/src/configurations/genie/synapseConfigs/dataReleases.ts:21]: New generic SynapseCompone
Screen Shot 2024-03-14 at 2 45 33 PM
nt collapse wrapper.

File: packages/synapse-react-client/src/components/Icon/Icon.tsx [hunk 4]
```diff
@@ -98,6 +98,7 @@ export type IconProps = {
 const iconSvgMapping = {
   clipboard: <IconSvg icon="clipboard" />,
   [DATASET]: <IconSvg icon="dataset" />,
+  [EXTERNAL_RESOURCE]: <IconSvg icon="link" />,
```
Comment (jay-hodgson) [inline packages/synapse-react-client/src/components/Icon/Icon.tsx:101]: Adam also requested we change the icon for the External Resources:

Screen Shot 2024-03-14 at 2 45 58 PM

File: packages/synapse-react-client/src/style/components/_cards.scss [hunk 5]
```diff
@@ -58,8 +58,13 @@ $text-color-lighter: #898989;
     @media (max-width: map-get(SRC.$breakpoints, 'medium')) {
       padding-top: 89px;
     }
-    .SRC-type,
+    .SRC-type {
+      color: inherit;
+    }
     .SRC-cardThumbnail {
```
Comment (jay-hodgson) [inline packages/synapse-react-client/src/style/components/_cards.scss:64]: I noticed that in all of the designed, the icon is centered vertically when not on a details page (when it's not a header card)

File: apps/portals/src/portal-components/SynapseComponentCollapse.tsx [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+import { Box, Collapse, SxProps, Typography } from '@mui/material'
+import {
+  KeyboardArrowDownTwoTone,
+  KeyboardArrowUpTwoTone,
+} from '@mui/icons-material'
+import { SynapseComponent } from '../SynapseComponent'
+
+export type SynapseComponentCollapseProps = {
+  text: string
+  synapseConfig: SynapseConfig
```
Comment (jay-hodgson) [inline apps/portals/src/portal-components/SynapseComponentCollapse.tsx:12]: Most of this file is styling.  Has state to know if it is collapsed or not, and uses a MUI Collapse and Sage SynapseComponent to do the heavy lifting

File: apps/portals/src/portal-components/SynapseComponentCollapse.tsx [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+            },
+          }}
+        >
+          {text}{' '}
+          {show ? (
+            <KeyboardArrowUpTwoTone sx={iconSx} />
+          ) : (
+            <KeyboardArrowDownTwoTone sx={iconSx} />
+          )}
+        </Typography>
```
Comment (hallieswan) [inline apps/portals/src/portal-components/SynapseComponentCollapse.tsx:60]: Looks good! Out of curiosity, how did you decide on using `Typography` (vs something like a `Button` with an `endIcon`)?

**Review by hallieswan — APPROVED** [review]
Looks good to me!

**Pull request merged**